packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/Examples.6/RowSelection.jsx

```diff
@@ -69,24 +69,26 @@ const RowSelection = () => {
   }, [selectedRow]);
 
   const onRowSelect = useCallback((rowToSelect) => {
-    const { rowId } = rowToSelect; // Contains rowId and sectionId
+    const { rowId, isMetaPressed } = rowToSelect; // Contains rowId and sectionId
     if (rowId) {
       const rowIndex = rowData.findIndex(e => e.id === rowId);
 
       const newRowData = [...rowData];
 
       // Remove current selections
       for (let row = 0; row < rowData.length; row += 1) {
-        newRowData[row].isSelected = false;
+        if (!isMetaPressed) newRowData[row].isSelected = false;
```

Comment:
nit: for better readablity
```suggestion
        if (!isMetaPressed) {
          newRowData[row].isSelected = false;
       }
```